src/mono/wasm/runtime/invoke-js.ts

```diff
@@ -94,37 +98,49 @@ export function mono_wasm_bind_js_function(function_name: MonoStringRef, module_
 function bind_fn_0V(closure: BindingClosure) {
     const fn = closure.fn;
     (<any>closure) = null;
+    const fqn = closure.fqn;
```

Comment:
how does this work? line 100 sets closure to `null` and then line 101 tries to get the `fqn` property from it.